### PR TITLE
docs: update screenshots in the UG, add manual testing information for clear and reset commands, add MSS for reset command

### PR DESCRIPTION
**Pull request by blaketingyu**
docs: update screenshots in the UG, add manual testing information for clear and reset commands, add MSS for reset command

### CODE REVIEW

File: docs/team/blaketingyu.md [hunk 1]
```diff
@@ -8,48 +8,50 @@ title: Blake Tan Ting Yu's Project Portfolio Page
 McKitchen (My-CLI-Kitchen) is a desktop application that can quickly store, edit, and search for recipes by running a single command. This is ideal for the fast typists, amateur cooks, busy students or executives who want a simple way to store or search for simple recipes to cook.
 The user interacts with it using a CLI-like interface, and it has a GUI created with JavaFX. It is written in Java and has about (to update) kLoC.
 
-Given below are my contributions to the project.
+**Summary of contributions**
 
-* **New Feature** Added `Storage` for `Recipe` class,and its respective fields (Pull request [#44](https://github.com/AY2122S2-CS2103T-T17-2/tp/pull/44))
+* **Code contributed:** : [RepoSense link (https://nus-cs2103-ay2122s2.github.io/tp-dashboard/?search=blaketingyu&brekdown=true)]()
```
Comment (yusufaine) [inline docs/team/blaketingyu.md:13]:
```suggestion
* **Code contributed:** : [RepoSense link](https://nus-cs2103-ay2122s2.github.io/tp-dashboard/?search=blaketingyu&brekdown=true)
```

**Review by yusufaine — APPROVED** [review]
LGTM, just gotta add a new line at the end of our DG 🌮

**Comment by blaketingyu** [comment]
> LGTM, just gotta add a new line at the end of our DG to pass the CI 🌮

Thanks bruh 🌮

File: docs/team/blaketingyu.md [hunk 2]
```diff
@@ -6,50 +6,55 @@ title: Blake Tan Ting Yu's Project Portfolio Page
 ### Project: McKitchen
 
 McKitchen (My-CLI-Kitchen) is a desktop application that can quickly store, edit, and search for recipes by running a single command. This is ideal for the fast typists, amateur cooks, busy students or executives who want a simple way to store or search for simple recipes to cook.
-The user interacts with it using a CLI-like interface, and it has a GUI created with JavaFX. It is written in Java and has about (to update) kLoC.
+The user interacts with it using a CLI-like interface, and it has a GUI created with JavaFX. It is written in Java and has about 10.3 kLoC.
```
Comment (gigituang) [inline docs/team/blaketingyu.md:9]:
```suggestion
The user interacts with it using a CLI-like interface, and it has a GUI created with JavaFX. It is written in Java and has about 8 kLoC.
```

**Review by gigituang — APPROVED** [review]
LGTMMM!!! 🌮

**Pull request merged**